src/Security/Member.php

```diff
@@ -1123,6 +1123,12 @@ public static function set_title_columns($columns, $sep = ' ')
 
     //------------------- HELPER METHODS -----------------------------------//
 
+    /* a method to allow using $LastName as an alternative to $Surname in templates since it is a commonly made mistake */
```

Comment:
Could you please update this docblock to follow the usual format as seen in other methods?